### PR TITLE
Local run tweaks

### PR DESCRIPTION
**Pull request by alongreyber**
Added a couple small changes to make it easier to debug a query with pdb:

- Enabled python-dotenv
- Added run.py file

### CODE REVIEW

File: strider/storage.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 from .config import settings
 
-if settings.redis_url == 'redis://fakeredis':
+if settings.redis_url.startswith('redis://fakeredis'):
```
Comment (patrickkwang) [inline strider/storage.py:9]: Is this change related to the others?

File: run.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import uvicorn
```
Comment (patrickkwang) [inline run.py:1]: I usually use a `main.sh` file to do this:
```bash
#!/usr/bin/env bash

export $(egrep -v '^#' .env | xargs)

# run api server
uvicorn strider.server:APP --host 0.0.0.0 --port 5781 --reload
```
I think I just prefer executables to be bash scripts than Python scripts, but are there other pros/cons to doing it either way?

Comment (alongreyber) [reply run.py:1]: The reason for the `run.py` script is so that it can be run with pdb. You can't do `python -m pdb main.sh` but you can do that with `run.py`. This is how FastAPI recommends debugging their applications: https://fastapi.tiangolo.com/tutorial/debugging/

File: strider/storage.py (hunk 1 above)
Comment (alongreyber) [reply strider/storage.py:9]: Kind-of. When reading from a .env file Pydantic will add the port to a redis url, so "redis://fakeredis" becomes "redis://fakeredis:6379". That breaks this check here, so the change is necessary to make this PR work. Another alternative way to do this could be to add the port to this string.

Comment (patrickkwang) [reply strider/storage.py:9]: Okay, let's make this more explicit by adding the port or, if for some reason the port might change, doing e.g. `re.fullmatch(f"redis://fakeredis:\d+", settings.redis_url)`

**Review by patrickkwang — CHANGES_REQUESTED** [review]

**Review by patrickkwang — APPROVED** [review]
